modules/game/client/controllers/game.client.controller.js

```diff
@@ -18,18 +18,29 @@ angular.module('game').controller('GameController', ['$scope', '$location', '$do
     $scope.CanvasSettings = CanvasSettings;
     $scope.ChatSettings = ChatSettings;
     $scope.GameSettings = GameSettings;
-    $scope.TopicSettings = TopicSettings;
     $scope.Object = Object;
 
+    // Get a list of topics from the server
+    $http.get('/api/topics').success(function (response) {
+      console.log('Retrieved list of topics', response);
+      $scope.listOfTopics = response;
+    }).error(function (response) {
+      console.error('Failed to fetch topics', response);
+    });
+
     // Pregame settings for host to change
     $scope.chosenSettings = {
       numRounds : GameSettings.numRounds.default,
       roundTime : GameSettings.roundTime.default,
       timeAfterGuess : GameSettings.timeAfterGuess.default,
-      topicListName: TopicSettings.topicListName.default,
-      topicListDifficulty: TopicSettings.topicListDifficulty.default
+      topicListName: undefined,
     };
 
+    // Fire off settings change events when the topicListName changes from the select dropdown
+    $scope.$watch('chosenSettings.topicListName', function() {
+      $scope.changeSetting('topicListName', $scope.chosenSettings.topicListName);
+    });
+
```

Comment:
We might want some refactoring around this in general.

Currently we use websocket messages to synchronize game state between clients. Maybe we should have some kind of "global watch" that watches all variables, and automatically sends websocket messages as they get changed.

This kind of `$watch` wasn't necessary before because we had `onClick` handlers instead. With a dropdown list we now need to watch for changes.